packages/sil-governance/src/routes/dashboard/index.stories.tsx

```diff
@@ -1,15 +1,13 @@
 import { storiesOf } from "@storybook/react";
 import React from "react";
 import { ReadonlyDate } from "readonly-date";
-import { DeepPartial } from "redux";
 
 import { ElectionFilter } from "../../components/AsideFilter";
 import { adminAddress, getDummyElectorates, getDummyProposalsState } from "../../store/proposals/dummyData";
-import { RootState } from "../../store/reducers";
 import DecoratedStorybook, { silRoot } from "../../utils/storybook";
 import Dashboard from "./index";
 
-const storeProps: DeepPartial<RootState> = {
+const storeProps: any = {
```

Comment:
@albertandrejev do you have any solution for this? I've seen you use `DeepPartial` in the Wallet without issue. Both the wallet and governance show errors on VS Code, but in Governance my `yarn start` fails because of that DeepPartial. Did you configure something so it's ignored or what?

Reply:
Interesting, `yarn start` should not compile `stories` files. But I didn't change anything.